app/src/main/java/sun/trainingcourse/anticovid_19/data/resource/CovidRepository.kt

```diff
@@ -1,5 +1,15 @@
 package sun.trainingcourse.anticovid_19.data.resource
 
-class CovidRepository : CovidDataSource.Local, CovidDataSource.Remote {
+class CovidRepository private constructor(
+    private val remote: CovidDataSource.Remote
+): CovidDataSource.Local, CovidDataSource.Remote {
 
+    override fun getCovidInformation() {
+    }
```

Comment:
```suggestion
    override fun getCovidInformation(): SomeType = TODO()
```
vì `TODO()` trả về Nothing